sdk/all/src/test/scala/org/typelevel/otel4s/sdk/OpenTelemetrySdkSuite.scala

```diff
@@ -56,7 +56,7 @@ class OpenTelemetrySdkSuite extends CatsEffectSuite {
       "OpenTelemetrySdk{" +
       "meterProvider=MeterProvider.Noop, " +
       "tracerProvider=TracerProvider.Noop, " +
-      s"propagators=ContextPropagators.Noop}, resource=${TelemetryResource.empty}}"
+      s"propagators=ContextPropagators.Noop}, resource=${TelemetryResource.default}}"
```

Comment:
should the noop one actually be empty instead, since it does nothing?

Reply:
Good catch. It should be `empty` indeed.